src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java

```diff
@@ -27,6 +27,75 @@ public CoffeeMaker() {
         deliveryPipe = new DeliveryPipe();
     }
 
+    /**
+     * Fill the boiler with cups of water
+     */
+    public void fillBoilerWithWater(final int cupsWater) {
+        boiler.setCupsOfWater(cupsWater);
+    }
+
+    /**
+     * Fill the boiler with cups of water
+     */
+    public void fillFilterWithCoffeeGrains() {
+        filterAndReceptacle.putCoffeeGrains();
+    }
+
+    /**
+     * Returns true if the pot is it's place
+     * @return boolean
+     */
+    public boolean obtainsStartButtonState() {
+        return startButton.getButtonPressed();
+    }
+
+    /**
+     * Returns true if the pot is it's place
+     * @return boolean
+     */
+    public void pressStartButton() {
+        startButton.isPressed();
```

Comment:
The name of this method seems to ask but it is not a boolean

Reply:
The name of the method asserts the action of pressing the button, is not assking if the button is pressed

Reply:
Ok I'll review the method name and change it